src/components/ttt-tile/ttt-tile.jsx

```diff
@@ -44,10 +54,7 @@ var TTTTile = React.createClass({
                     </div>
 
                 </a>
-                <div className="ttt-tile-guides" onClick={this.props.onGuideClick}>
-                    <FormattedMessage id='tile.guides' defaultMessage='See Cards and Guides'/>
-                    <img className="ttt-tile-see-more" src="/svgs/ttt/see-more.svg" />
-                </div>
+                {this.tileGuidesIfExists()}
```

Comment:
I personally find inline rendering easier to understand since all the logic is in one place. Here I would either use the pattern
```jsx
{this.props.onGuideClick && (
    <div ...
)}
```
or add a component to represent the tile guide section and have it only return a component if it receives the `onClick` prop.

If this is based on a pattern that already exists on www then you can ignore that comment :)